AVFoundation-CombineTests/TestSubscriber.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TestSubscriber.swift
+//  AVFoundation-CombineTests
+//
+//  Created by József Vesza on 2020. 07. 24..
+//  Copyright © 2020. József Vesza. All rights reserved.
+//
+
+import Combine
+
+class TestSubscriber<T>: Subscriber {
+    typealias Input = T
+    typealias Failure = Never
+    
+    let demand: Int
```

Comment:
@jozsef-vesza  Now i'm wondering ... could it be better that this `demand` is of type `Demand` instead of `Int`? that way you could also initialize it as **unlimited** and then you wouldn't need to use `sink` in your tests and be more explicit about what is being tested in those cases.